### PR TITLE
fix: select editing

### LINKED ISSUE #200: After selecting multiple lines, editing only modifies the first line
After selecting multiple lines, editing only modifies the first line
delete

### PR DESCRIPTION
**Pull request by himself65**
Fixes: https://github.com/toeverything/blocksuite/issues/200

### CODE REVIEW

File: packages/blocks/src/page-block/default/default-page-block.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ import {
   getCurrentRange,
   isMultiBlockRange,
   getModelsByRange,
-} from '../../__internal__'
+} from '../../__internal__';
```
Comment (doodlewind) [inline packages/blocks/src/page-block/default/default-page-block.ts:37]: Just saying that we have `pnpm format` for prettifying.

**Review by doodlewind — APPROVED** [review]
Great fix, thanks!

cc @lawvs

File: packages/blocks/src/page-block/default/default-page-block.ts (hunk 1 above)
Comment (himself65) [reply packages/blocks/src/page-block/default/default-page-block.ts:37]: I would like to use [husky](https://www.npmjs.com/package/husky) to run it automatically before submitting

**Pull request merged**